openbb_terminal/stocks/options/options_view.py

```diff
@@ -473,37 +474,30 @@ def display_chains(
         Format for exporting data
     sheet_name: str
         Optionally specify the name of the sheet to export to
-    to_display: list
-        List of columns to display
     """
-    if to_display:
-        to_display += ["strike", "optionType"]
-        to_display = list(set(to_display))
-        chain = chain[to_display]
-
     min_strike, max_strike = get_strikes(
         min_sp=min_sp, max_sp=max_sp, current_price=current_price
     )
 
     chain = chain[chain["strike"] >= min_strike]
     chain = chain[chain["strike"] <= max_strike]
     calls, puts = get_calls_and_puts(chain)
-
-    chain = get_greeks(
-        current_price=current_price,
-        calls=calls,
-        expire=expire,
-        puts=puts,
-        show_all=True,
-    )
-
-    # if the greeks calculation went with no problems, otherwise keep the previous
-    if not chain.empty:
-        calls, puts = get_calls_and_puts(chain)
-        console.print("Greeks calculated by OpenBB.")
-
-    calls = calls.sort_index(axis=1)
-    puts = puts.sort_index(axis=1)
+    # Tradier provides the greeks, so calculate them if they are not present
+    if "delta" not in chain.columns:
```

Comment:
So this was my debate above.  Since this endpoint only provides one greek, I made the decision to show what they provide instead of our very simple calculation.

Reply:
I'm good with this!